### ISSUE TITLE
Starting the ResBaz 2017 website

### ISSUE THREAD

**Issue by DamienIrving** [resbaz/resbaz2017#1]
@BillMills @tjdett I've taken the core elements from the 2016 site and put them in this new repo for the ResBaz 2017 website. 

I'm thinking resbaz2017.com this time around?

The first road block we are going to face (I'm dealing with it right now!) is updating the Jekyll installation instructions in the README. It seems OS X El Capitan is making things difficult:
http://stackoverflow.com/questions/32891965/error-while-executing-gem-errnoeperm-operation-not-permitted

**Comment by tjdett** [comment]
@DamienIrving how about feb2017.resbaz.com or q1-2017.resbaz.com? Buying and maintaining a new domain for each year seems excessive.

**Comment by DamienIrving** [comment]
@tjdett Good point. I'd be happy with 2017.resbaz.com? The way things played out this year is that we used the same website for the entire year, regardless of whether people hosted their event during the first week of February or not. I can't see that changing next year, so one site for the whole of 2017 should be fine.

**Comment by DamienIrving** [comment]
@BillMills I've added Hobart and Melbourne pages as examples, but I'm having trouble previewing them. `http://127.0.0.1:4000` works fine (after typing `jekyll serve --future` in the base directory for the repo), but `http://127.0.0.1:4000/melbourne` and `http://127.0.0.1:4000/hobart` don't. Do you know why that is?

**Comment by tjdett** [comment]
I had to double check that all-numeric subdomains are allowed. Answer: [yes, since 1989](http://serverfault.com/a/516230/78947).

`2017.resbaz.com` should be fine.

**Comment by tjdett** [comment]
@DamienIrving https://2017.resbaz.com/ is now live.

**Comment by DamienIrving** [comment]
@tjdett Fantastic! Thanks.

Do you know why https://2017.resbaz.com/melbourne and https://2017.resbaz.com/hobart don't work?

**Comment by DamienIrving** [comment]
@tjdett Thanks for those pull requests.

Is https://2016-02-01.resbaz.com/ now dead? There's a chance one more site will run a ResBaz in 2016, so I'm thinking we should leave that site up for the moment and hold off on re-directing resbaz.com to the 2017 site until we know there aren't going to be anymore for 2016.

**Comment by DamienIrving** [comment]
Ah, I see the issue. resbaz.com directs to 2016-02-01.resbaz.com when it should go to https://feb2016.resbaz.com/
